project1.html

```diff
@@ -1,121 +1,205 @@
+          <div class="flex-container">
+            <div class="flex-container3">
+              <div>
+                <a><img src="Ellipse1.jpg" alt="Javascript" /></a>
+              </div>
+              <div><h5>Javascript</h5></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+            </div>
+            <div class="flex-container4">
+              <div>
+                <a><img src="Ellipse1(1).jpg" alt="HTML" /></a>
+              </div>
+              <div><h5>HTML</h5></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+            </div>
+            <div class="flex-container5">
+              <div>
+                <a><img src="Ellipse1(2).jpg" alt="CSS" /></a>
+              </div>
+              <div><h5>CSS</h5></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+            </div>
+          </div>
+          <div>
+            <a class="Vector1"><img src="Vector1.png" alt="" /></a>
+            <a class="Vector2"><img src="Vector2.png" alt="" /></a>
+            <a class="Vector3"><img src="Vector2.png" alt="" /></a>
+            <div><h4 class="Frameworks">Frameworks</h4></div>
+            <a class="Separator1"><img src="SeparatorBottom.jpg" alt="" /></a>
+          </div>
+          <div><h4 class="Skills">Skills</h4></div>
+          <a class="Separator2"><img src="SeparatorBottom.jpg" alt="" /></a>
+          <a class="Separator1"><img src="SeparatorBottom.jpg" alt="" /></a>
+        </div>
+
+        <div class="add-contact add-contact-bckgrnd">
+          <div class="image"></div>
+          >
+
+          <h1>Contact me</h1>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+
+          <form
+            action="https://formspree.io/f/xzboleol"
+            method="POST"
+            class="form"
+          >
+            <ul>
+              <li>
+                <input
+                  type="text"
+                  id="name"
+                  name="name"
+                  placeholder="contact name"
+                  maxlength="30"
+                />
+              </li>
+              <li>
+                <input
+                  type="email"
+                  id="mail"
+                  name="email"
+                  placeholder="contact@gmail.com"
+                />
+              </li>
+              <li>
+                <textarea
+                  id="msg"
+                  name="message"
+                  placeholder="Write your message here"
+                  maxlength="500"
+                ></textarea>
```

Comment:
- [ ] `required`
- [ ] please make sure All input fields have the required attribute to activate the validation on submit